e2e/command_test.go

```diff
@@ -147,7 +252,144 @@ func TestE2E_EnableTaskCommand(t *testing.T) {
 	}
 }
 
-// TestE2E_DisableTaskCommand tests the CLI to disable an enabled task. This
+// TestE2E_EnableTaskCommand tests the task Enable CLI and confirms the expected
+// output and state given different paths. This test starts up a local Consul server
+// and runs CTS with TLS enabled, with a disabled task.
+func TestE2E_EnableTaskCommand_TLS(t *testing.T) {
```

Comment:
Totally down for testing TLS on our existing enable and disable commands, but wanted to start thinking about what our long term strategy is for testing TLS with our CLI. We'll be eventually adding more CLI commands, do we want to always be having test versions for both TLS enabled and disabled?

Doesn't have to be in this PR, but we could have a separate test or test suite that checks all these combinations on all our commands just to see that the CLI is connecting and not necessarily checking the behavior of the command. Then we wouldn't have to always include the TLS test versions for the functionality tests.

Reply:
Talked to Melissa about this, I agree that TLS should just be checking the connection was successful. I will refactor this at a later date